--- xmlschema-1.10.0/xmlschema/testing/__init__.py.orig	2022-03-07 14:26:41.000000000 +0100
+++ xmlschema-1.10.0/xmlschema/testing/__init__.py	2022-03-13 07:47:58.128765791 +0100
@@ -18,6 +18,7 @@ that contain a list of relative paths to
 build a set of test classes. Each path is followed by a list of options that defines a
 custom setting for each test.
 """
+import os
 from urllib.request import urlopen
 from urllib.error import URLError
 
@@ -40,7 +41,7 @@ def has_network_access(*locations):
     return False
 
 
-SKIP_REMOTE_TESTS = not has_network_access(
+SKIP_REMOTE_TESTS = os.environ.get("XMLSCHEMA_SKIP_REMOTE_TESTS", 0) or not has_network_access(
     'https://github.com/', 'https://www.w3.org/', 'https://www.sissa.it/'
 )
 
